### PR TITLE
Added a hide UI hotkey

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/6779.

### CODE REVIEW

File: OpenRA.Game/Widgets/RootWidget.cs [hunk 1]
```diff
@@ -31,6 +31,14 @@ public override bool HandleKeyPress(KeyInput e)
 					return true;
 				}
 
+				if (hk == Game.Settings.Keys.HideUserInterfaceKey)
+				{
+					foreach (var child in this.Children)
+						child.Visible = child.Visible ^ true;
```
Comment (pchote) [inline OpenRA.Game/Widgets/RootWidget.cs]: Does toggling this widget's visibility not work?

In any case, `child.Visible ^= true`.

Comment (Mailaender) [inline OpenRA.Game/Widgets/RootWidget.cs]: Updated. It needs to be visible to listen for and fire up the un-hide hotkey event.

**Comment by abcdefg30** [comment]
Looks good to me :+1: / :white_check_mark:

**Comment by obrakmann** [comment]
Looks and works fine. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/f979a6975917a848aa4be618e5afbd32c066e3d1%5E...f979a6975917a848aa4be618e5afbd32c066e3d1)

**Pull request merged**